src/lib/outputProcessor/autogen/json.interp

```diff
@@ -0,0 +1,40 @@
+token literal names:
```

Comment:
These auto generated files will be created at runtime, and they are unable to be fine-grained controlled. Maybe it's more appropriate to add them in .gitignore.

Reply:
Thanks for the suggestion, I will do it.